### ISSUE TITLE
Infection Monkey can't find any victims

### ISSUE THREAD

**Issue by moonspell79** [guardicore/monkey#141]
I installed Infection Monkey from AWS Marketplace in public subnet along with a few other servers. One server has apache running and the other has SSH port open which allows to login using local user (defined in IM Configuration login/password to check). Unfortunatly the Infection Map does not show anything except machine where it is installed.

08/06/2018 14:29:03 ip-172-31-15-198: Monkey died.
08/06/2018 14:29:18 ip-172-31-15-198: Monkey started.
08/06/2018 14:29:18 ip-172-31-15-198: No tunnel is used.
08/06/2018 14:29:18 ip-172-31-15-198: Monkey collected system information.

What might be wrong?

**Comment by danielguardicore** [comment]
Hey, I'd suggest using the [Google Group](https://groups.google.com/forum/#!forum/infection-monkey) for these type of questions.

My first question would be 
1. Are the two machines in the same subnet?
2. Is the Monkey configured with `Local network scan` ? You can check this under `Basic -Network` and to make sure it's marked.

**Comment by TRGamer-tech** [comment]
Sorry for opening this issue again, but I am currently having the exact same problem.
Im working on a virtual windows 10 PC with the Island Server installed. 
All the devices (2x Windows Server 2019, 3x Windows 10 and 1x Windows 7) are in the same subnet and can be pinged. `Local Network Scan` is also enabled.
I tried searching, but couldn't find anything about this. And since the google group is not available anymore, I have to open this up again.
I also already tried entering the credentials and users to the config, which didn't work.

Is there anything else I could try to do?
